website/docs/sdk-reference/php.md

```diff
@@ -110,12 +119,129 @@ $client = new \ConfigCat\ConfigCatClient("#YOUR-SDK-KEY#");
 $keys = $client->getAllKeys();
 ```
 
+## Flag Overrides
+
+With flag overrides you can overwrite the feature flag & setting configuration fetched from the ConfigCat CDN with local values.
+Moreover, you can specify how the overrides should apply over the fetched configuration. The following 3 behaviours are supported:
+
+- **Local/Offline mode** (`OverrideBehaviour::LOCAL_ONLY`): With this mode, the SDK won't fetch feature flags & settings from the ConfigCat CDN and will use only the local overrides to evaluate feature flags. This mode is designed to support disonnected environments. You can use it in your development environment, automated tests, or isolated production machines.
+
+- **Local over remote** (`OverrideBehaviour::LOCAL_OVER_REMOTE`): With this mode, the SDK will fetch feature flags & settings from the ConfigCat CDN and will override those that have a matching key.
+
+- **Remote over local** (`OverrideBehaviour::REMOTE_OVER_LOCAL`): With this mode, the SDK will fetch feature flags & settings from the ConfigCat CDN and will use the overrides for those flags only that doesn't exist in the fetched configuration.
+
+You can load your feature flag & setting overrides from a file or from a simple associative array.
+
+### JSON File
+
+The SDK can be configured to load your feature flag & setting overrides from a file or classpath resource. 
+You can also specify whether the file should be reloaded when it gets modified.
+```php
+$client = new ConfigCatClient("localhost", [
+  ClientOptions::FLAG_OVERRIDES => OverrideDataSource::localFile(
+    "tests/test-rules.json", // path to the file
+    OverrideBehaviour::LOCAL_ONLY // local/offline mode
+  ),
+]);
+```
+#### JSON File Structure
+The SDK supports 2 types of JSON structures to describe feature flags & settings.
+
+##### 1. Simple (key-value) structure
+```json
+{
+  "flags": {
+    "enabledFeature": true,
+    "disabledFeature": false,
+    "intSetting": 5,
+    "doubleSetting": 3.14,
+    "stringSetting": "test"
+  }
+}
+```
+
+##### 2. Complex (full-featured) structure
+This is the same format that the SDK fetches from the ConfigCat CDN. 
+It allows the usage of all features you can do on the ConfigCat Dashboard.
```

Comment:
The URL would be helpful from where they can download their current config.json file so they can re-write it.

Reply:
done